src/sidebarLearn.json

```
@@ -86,36 +86,36 @@
       ]
     },
     {
-      "title": "Adding Interactivity",
+      "title": "Ajout d'interactivité",
       "path": "/learn/adding-interactivity",
       "tags": [],
       "routes": [
         {
-          "title": "Responding to Events",
+          "title": "Répondre aux événements",
           "path": "/learn/responding-to-events"
         },
         {
-          "title": "State: A Component's Memory",
+          "title": "État : la mémoire d'un composant",
           "path": "/learn/state-a-components-memory"
         },
         {
-          "title": "Render and Commit",
+          "title": "Le rendu et la mise à jour",
           "path": "/learn/render-and-commit"
         },
         {
-          "title": "State as a Snapshot",
+          "title": "L'état comme une capture",
           "path": "/learn/state-as-a-snapshot"
         },
         {
-          "title": "Queueing a Series of State Updates",
+          "title": "Mettre en file d'attente une série de mises à jour d'état",
```

```suggestion
          "title": "Cumuler les mises à jour d’un même état",
```

On parle ici de cumuler correctement des mises à jour d'une même varaible état (updater function plutôt que valeur). Cette formulation me semble un bon compromis entre concision et précision.